features/apps/src/main/feature/feature.xml

```diff
@@ -24,6 +24,7 @@
     <repository>mvn:ddf.features/solr/${project.version}/xml/features</repository>
     <repository>mvn:ddf.features/utilities/${project.version}/xml/features</repository>
     <repository>mvn:ddf.catalog.solr/catalog-solr-app/${project.version}/xml/features</repository>
+    <repository>mvn:ddf.catalog.solr/split-solr-app/${project.version}/xml/features</repository>
```

Comment:
❓No feature declarations are being used. Can this line be removed?

Reply:
it is used here by installed-profile, transient usage. https://github.com/lamhuy/ddf/blob/splitcatalog/features/install-profiles/src/main/feature/feature.xml#L19